### DISCUSSION TITLE
Tracker horizontal layout on Dashboard

### DISCUSSION THREAD

**Discussion by dkzsys** [TfTHacker/DashboardPlusPlus#30] [Q&A]
Hi there, I got the idea of putting my tracker chart on the dashboard from Sergio youtube vid. 

However, when I tried to implement it, I couldn't put tracker next to the bullet point. As a result, all tracker charts are listed vertically. 

I found a reddit post that asked this exact challenge.
https://www.reddit.com/r/ObsidianMD/comments/18amjqt/dashboard_lining_up_trackers_in_a_row/
![image](https://github.com/TfTHacker/DashboardPlusPlus/assets/39715311/2de68722-37a6-44b0-85cc-ef3b2f343805)

I tried using embedded content, not useful as the window size is restricted.

Any suggestion would be much appreciated. Thanks

**Comment by TfTHacker** [comment]
It is a great idea, however, the snippet used here with simple markdown list doesn't allow for too many fancy options. There are some dashboard solutions out there that use callouts, which allow for the layout of such components into bigger blocks.
